kuma/api/apps.py

```diff
@@ -12,5 +12,3 @@ class APIConfig(AppConfig):
 
     def ready(self):
```

Comment:
We should probably just get rid of the `ready` method altogether.

Reply:
done. Thanks!